activityDoc.md

```diff
@@ -297,23 +314,25 @@ Example:
   },
   target: {
     id: <publicationId>
+    type: <pubOrNote>
```

Comment:
you should have a different section for publication and note. Eventually they might be more different (for example accepted tag types might be different, which would have to be explained.